lib/tasks/export.rake

```diff
@@ -0,0 +1,125 @@
+require 'csv'
+
+namespace :export do
+  desc 'Export Firms to CSV=(/path/to/csv)'
+  task firms: :environment do
+    unless csv_path = ENV['CSV']
+      puts 'Usage: rake export:firms CSV=/path/to/csv.ext'
+      abort
+    end
+
+    CSV.open(csv_path, 'wb') do |csv|
+      csv << [
+        'ID',
+        'Parent ID',
+        'Created At',
+        'Updated At',
+        'FCA Number',
+        'Registered Name',
+        'Email Address',
+        'Telephone Number',
+        'Address Line One',
+        'Address Line Two',
+        'Address Town',
+        'Address County',
+        'Address Postcode',
+        'In-person Advice Methods',
+        'Other Advice Methods',
+        'Free Initial Meeting',
+        'Initial Meeting Duration',
+        'Initial Advice Fee Structures',
+        'Ongoing Advice Fee Structures',
+        'Allowed Payment Methods',
+        'Minimum Fixed Fee',
+        'Retirement Income Products Percent',
+        'Pension Transfer Percent',
+        'Long-Term Care Percent',
+        'Equity Release Percent',
+        'Inheritance Tax and Estate Planning Percent',
+        'Wills and Probate percent',
+        'Other Percent',
+        'Investment Sizes'
+      ]
+
+      Firm.order(id: :asc).each do |firm|
```

Comment:
Maybe include `where.not(email_address: nil)` to only dump those firms that completed the questionnaire? Remembering that partially entered firms are also in the database. If they want the partially entered firms in this dataset then obviously leave as-is.